lua-docs/content/reference/client.yml

```diff
@@ -85,21 +85,95 @@ properties:
       type: "Clouds"
       description: "See [Clouds]."
 
-    - name: "DirectionDidChange"
+    - name: "DirectionalPad"
       type: "function"
       description: |
-          Triggered when the direction changes. (using direction keys or direction gamepad)
-          The default implementation does not appear in sample scripts, but it can be overriden.
+          Triggered when what we call the "direction pad" or "D-pad" is activated. 
+
+          The D-pad maps to the direction keys when using a keyboard (WASD by default).
+
+          When using a touch screen, the D-pad is by default represented by the virtual pad that's on the left side of the screen.
+
+          The function receives 2 arguments `x` & `y` to represent 8 possible directions or the idle state:
+
+          `0,0 -> IDLE`
+          `0,1 -> UP`, `1,0 -> RIGHT`, `0,-1 -> DOWN`, `-1,0 -> LEFT`
```

Comment:
I would prefer to have the informations in a "loop":
`0,0 -> IDLE`
`0,1 -> UP`, `√2,√2 -> UP/RIGHT`, `1,0 -> RIGHT`, `√2,-√2 -> DOWN/RIGHT`, `0,-1 -> DOWN`, `-√2,-√2 -> DOWN/LEFT`, `-1,0 -> LEFT`, `-√2,√2 -> UP/LEFT`

Reply:
And maybe a quick explanation on why \sqrt(2), not everyone is familiar with trigonometry !

Reply:
Yes, we will add images for that, it worries me to add too much text.